### DISCUSSION TITLE
Limit number of "project management" and "helper" repositories

### DISCUSSION THREAD

**Discussion by SebastianBezold** [eclipse-tractusx/sig-infra#156] [General]
## Issue

We have a very fast growing number of repositories in eclipse-tractusx. 
It seems like a lot of them are "helper" or dedicated to "project management".
To me it is not really clear, if they are distinctive enough so that we actually need a dedicated repo, or if combining them in fewer repos with a proper directory structure would be sufficient.

Keeping track of what is happening where is increasingly hard. Especially for newcomers.

## What we could combine

I think we should combine project management and technical repos. Here a list of repos, that in my opinion are candidates to combine:

### technical + helper

- [sig-infra](https://github.com/eclipse-tractusx/sig-infra)
- [charts](https://github.com/eclipse-tractusx/charts)
- [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release)
- [tractusx-quality-checks](https://github.com/eclipse-tractusx/tractusx-quality-checks)
- [e2e-testing](https://github.com/eclipse-tractusx/e2e-testing)

### project management

- [community](https://github.com/eclipse-tractusx/community)
- [A new proposed one 'roadmap'](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3324)

**Comment by almadigabor** [comment]
Agreed. Too much repos, hard to determine which is responsible for what. I would maybe combine 

__sig-infra__

- [sig-infra](https://github.com/eclipse-tractusx/sig-infra)
- [tractus-quality-checks](https://github.com/eclipse-tractusx/tractusx-quality-checks)
- [e2e-testing](https://github.com/eclipse-tractusx/e2e-testing)

__release__

- [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release)
- [charts](https://github.com/eclipse-tractusx/charts)

__project management__

- [community](https://github.com/eclipse-tractusx/community)
- [A new proposed one 'roadmap'](https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/3324)

**Reply by FaGru3n** [reply]
update to project management

one repo remaining [sig-release](https://github.com/eclipse-tractusx/sig-release)

**Reply by FaGru3n** [reply]
__sig-infra__

- [sig-infra](https://github.com/eclipse-tractusx/sig-infra)
- [tractus-quality-checks](https://github.com/eclipse-tractusx/tractusx-quality-checks)
- [e2e-testing](https://github.com/eclipse-tractusx/e2e-testing)

__release + project management__

- [tractus-x-release](https://github.com/eclipse-tractusx/tractus-x-release)
- [charts](https://github.com/eclipse-tractusx/charts)
- [sig-release](https://github.com/eclipse-tractusx/sig-release)
